### PR TITLE
OSS Ad for #861

### LINKED ISSUE #861: Display Ads that support Open Source
A conversation with @piamancini

> We've been talking to CarbonAds about the [thread we started on twitter](https://twitter.com/_prateekbh/status/881806838253146112) and we are running a test with the people who showed interest in the survey.

> We've created an ad code for https://www.bootstrapcdn.com/ and the monthly payouts will go to the Open Source Collective. The script  you need to include is: 

```html
<script async type="text/javascript" src="//cdn.carbonads.com/carbon.js?zoneid=1673&serve=C6AILKT&placement=bootstrapcdncom" id="_carbonads_js"></script>
```

> Please keep in mind that it is possible for the styling of the ad to inherit other styles from your site, and you may need to add some additional styles to make it look perfect .Let me know if you want an ad placement suggestion for your site with some CSS. 

* [Display Networks - FAQ - Publishers](https://www.evernote.com/shard/s456/sh/1b37bdf5-937f-4b8e-a480-d3bdef146b7d/19f65f820e274f6234149eb7ce105dad)
* [Carbon Ad Style Guide](https://www.evernote.com/shard/s456/sh/62b7c21b-fd20-46f7-84ec-80f9b5a0892e/81cfa6e35a6cbc6c6f9af47c19d874a7)

@jdorfman's Todos:

- [x] Talk to James L at StackPath
- [x] Create PR for @XhmikosR #862 
- [ ] Follow up with Pia

### PR DESCRIPTION
**Pull request by jdorfman**
~~~CSS and JS is added but doesn't show up despite being [generated](https://cl.ly/3p0n3v3l3U3u).~~~

~~~Help.~~~

TODO:

- [x] Verify that the current CSP in this PR is right and the ads work properly
- [x] Add link to modal explaining ad
- [ ] Move script tag to the bottom and initialize it manually; I need more docs on this

Fixes #861

### CODE REVIEW

**Comment by XhmikosR** [comment]
I'll have a closer look when I'm back at my dev machine. Please revert the
package lock removal.

**Comment by XhmikosR** [comment]
@jdorfman: still not back at home to fix this, but are you sure about the place you put the ad? IIRC it must be a lot higher in the page.

**Comment by jdorfman** [comment]
@XhmikosR reverted

**Comment by XhmikosR** [comment]
Please have a look at the patches I'll push; it would be nice if I didn't have to do the same each time you push something :)

Now, regarding this branch. I'm pretty sure where you placed the ad isn't compliant with the rules. Please read those one more time.

And do not merge until I'm done with this branch, please :)

**Comment by XhmikosR** [comment]
Things left:

- [ ] Verify that the current CSP in this PR is right and the ads work properly
- [ ] Move script tag to the bottom and initialize it manually; I need more docs on this

**Comment by jdorfman** [comment]
@XhmikosR I kind of like the placement of the ad. I think maybe we try it there first and move it up if it isn't performing great.

Side note for me: blog post (or modal) explaining what the ad is doing.

File: views/_partials/footer.pug [hunk 1]
```diff
@@ -24,6 +24,6 @@ footer
         |  &amp; 
         a(href=config.friends[4].url, target='_blank', rel='noopener')=config.friends[4].name
 
-    script(async, src='https://cdn.carbonads.com/carbon.js?zoneid=1673&serve=C6AILKT&placement=bootstrapcdncom' id='_carbonads_js')
+    script#_carbonads_js(async, src='https://cdn.carbonads.com/carbon.js?zoneid=1673&serve=C6AILKT&placement=bootstrapcdncom')
```
Comment (jdorfman) [inline views/_partials/footer.pug]: that would explain it

**Comment by jmervine** [comment]
👍

**Comment by XhmikosR** [comment]
I added the modal but I don't like it. I feel like a tooltip might be better, but I'm not sure how it will behave on mobile...

**Comment by jdorfman** [comment]
@XhmikosR are we ready to deploy?

**Comment by XhmikosR** [comment]
Please check your slack. And never hurry to merge because issues will arise.

**Comment by jdorfman** [comment]
Merging this tomorrow morning. cc @XhmikosR

**Comment by XhmikosR** [comment]
@jdorfman: I honestly do not see the hurry. You assigned the issue to me, correct? Then wait until I have the time to sort it.

**Comment by jdorfman** [comment]
🤘

**Pull request merged**